infra/aws/terraform/iam/eks-e2e-boskos/cloudformation-user.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Resources": {
+    "provisioner": {
+      "Type": "AWS::IAM::User",
+      "Properties": {
+        "Path": "/",
+        "UserName": "provisioner",
+        "Policies": [
+          {
+            "PolicyName": "ProvisionerUserRole",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Action": "sts:AssumeRole",
+                  "Effect": "Allow",
+                  "Resource": "*"
```

Comment:
This seems quite broad. I guess we're relying on role trust policies here. Is that safe?

Reply:
I agree with that and I did this instead:

```yaml
              - Action: 'sts:AssumeRole'
                Effect: Allow
                Resource: 'arn:aws:iam::*:role/Provisioner'
```

Is this any better?